DataConnectors/SquadraTechnologies.SecRMM.json

```diff
@@ -0,0 +1,119 @@
+      "query": "secRMM_CL | where ((Event_s == 'EXTERNAL') and (Message contains 'Microsoft Defender'))"
+    },
+    {
+      "description": "Hardware Encrypted Device Activity",
+      "query": "secRMM_CL | where DeviceDescription_s contains ' ENCRYPTED Removable Media'"
+    },
+    {
+      "description": "Users who have tried to execute macros or programs from a removable storage device",
+      "query": "secRMM_CL | where ((Event_s == 'BLOCK MACROS ON DEVICE ACTIVE') or (Event_s == 'BLOCK PROGRAMS ON DEVICE ACTIVE')) | summarize count() by User_s"
+    },
+    {
+      "description": "Removable storage devices that are not encrypted (hardware or software)",
+      "query": "secRMM_CL | where ((Event_s == 'ONLINE') and (DeviceDescription_s !contains 'ENCRYPTED'))"
+    },
+    {
+      "description": "Removable storage devices that are mounted into a Virtual Machine (from physical)",
+      "query": "secRMM_CL | where ((Event_s == 'EXTERNAL') and (Message contains 'ONLINE') and (Message contains 'Virtual Machine'))"
+    },
+    {
+      "description": "Removable storage devices that are mounted into a Virtual Machine (from virtual)",
+      "query": "secRMM_CL | where ((Event_s == 'ONLINE') and (Drive_s contains '^'))"
+    },
+    {
+      "description": "Show mobile devices that are being USB mounted",
+      "query": "secRMM_CL | where ((Event_s == 'ONLINE') and (DeviceDescription_s contains 'MOBILE'))"
+    },
+    {
+      "description": "Show mobile devices that are being USB mounted but are not MDM (Microsoft Intune) enrolled",
+      "query": "secRMM_CL | where ((Event_s == 'ONLINE') and (AdditionalProgramInfo_s contains 'Mobile device is not MDM enrolled'))"
+    }
+  ],
+  "dataTypes": [
+    {
+      "name": "secRMM_CL",
+      "lastDataReceivedQuery": "secRMM_CL | summarize Time = max(TimeGenerated) | where isnotempty(Time)"
+    }
+  ],
+  "connectivityCriterias": [
+    {
+      "type": "IsConnectedQuery",
+      "value": [
+        "secRMM_CL | summarize LastLogReceived = max(TimeGenerated) | project IsConnected = LastLogReceived > ago(30d)"
+      ]
+    }
+  ],
+  "availability": {
+    "status": 1
+  },
+  "permissions": {
+    "resourceProvider": [
+      {
+        "provider": "Microsoft.OperationalInsights/workspaces",
+        "permissionsDisplayText": "read and write permissions are required.",
+        "providerDisplayName": "Workspace",
+        "scope": "Workspace",
+        "requiredPermissions": {
+          "write": true,
+          "read": true,
+          "delete": true
+        }
+      }
+    ],
+    "customs": [
+    ]
+  },
+  "instructionSteps": [
+    {
+      "title": "",
+      "description": "Follow the step-by-step instructions provided in the 'Squadra Technologies configuration guide for Azure Sentinel' at http://www.squadratechnologies.com/StaticContent/ProductDownload/secRMM/9.9.0.0/secRMMAzureSentinelAdministratorGuide.pdf",
```

Comment:
Please make Squadra Technologies configuration guide for Azure Sentinel as a clickable link to the PDF guide and please remove the long URL to the PDF guide. You can remove the quotes from 'Squadra Technologies configuration guide for Azure Sentinel' as well

Reply:
Hi, OK, fixed.